.extlib/simplesamlphp/modules/saml/src/Auth/Source/SP.php

```diff
@@ -306,6 +306,22 @@ public function getHostedMetadata(): array
      */
     public function getIdPMetadata(string $entityId): Configuration
     {
+        assert(is_string($entityId));
+
+        global $saml2auth;
```

Comment:
Can we get a modification comment highlighting this as a auth_saml2 modification